og_migrate/tests/src/Kernel/Migrate/OgUserMembershipD7MigrateTest.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace Drupal\Tests\og_migrate\Kernel\Migrate;
+
+use Drupal\og\Entity\OgMembership;
+use Drupal\og\OgMembershipInterface;
+use Drupal\Tests\migrate_drupal\Kernel\d7\MigrateDrupal7TestBase;
+
+/**
+ * Tests organic group membership migration for users.
+ *
+ * @group og_migrate
+ *
+ * @internal
+ */
+class OgUserMembershipD7MigrateTest extends MigrateDrupal7TestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
+    'text',
+    'entity_reference',
+    'telephone',
+    'datetime',
+    'image',
+    'link',
+    'filter',
+    'menu_ui',
+    'comment',
+    'node',
+    'forum',
+    'system',
+    'taxonomy',
+    'user',
+    'og',
+    'og_ui',
+    'og_migrate',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp(): void {
+    parent::setUp();
+
+    $this->loadFixture(__DIR__ . '/../../../fixtures/drupal7.php');
+    $this->installEntitySchema('og_membership_type');
+    $this->installEntitySchema('og_membership');
+    $this->installEntitySchema('og_role');
+    $this->installEntitySchema('user');
+    $this->installEntitySchema('node');
+    $this->installEntitySchema('comment');
+    $this->installEntitySchema('comment_type');
+    $this->installEntitySchema('taxonomy_term');
+    $this->installEntitySchema('taxonomy_vocabulary');
+    $this->installConfig(['comment', 'user', 'node', 'og']);
+    $this->installSchema('comment', ['comment_entity_statistics']);
+    $this->installSchema('forum', ['forum_index', 'forum']);
+    $this->installSchema('node', ['node_access']);
+    $this->executeMigrations([
+      'd7_filter_format',
+      'd7_user_role',
+      'd7_user',
+      'd7_node_type',
+      'd7_node:test_content_type',
+      'd7_node:article',
+      'd7_node:page',
+      'd7_node:forum',
+      'd7_node:blog',
```

Comment:
Migration d7_comment did not meet the requirements. Missing migrations d7_node:et. requirements: d7_node:et.

```suggestion
      'd7_node:blog',
      'd7_node:et',
```